pkg/kubelet/apis/config/types.go

```diff
@@ -326,7 +326,7 @@ type KubeletConfiguration struct {
 	IPTablesDropBit int32
 	// featureGates is a map of feature names to bools that enable or disable alpha/experimental
 	// features. This field modifies piecemeal the built-in default values from
-	// "k8s.io/kubernetes/pkg/features/kube_features.go".
+	// "https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go".
```

Comment:
This corrected link is not right. This isn't a link, it's the name to the fully qualified golang package.